src/main/groovy/edu/oregonstate/mist/webapiskeleton/SkeletonApplication.groovy

```diff
@@ -25,17 +27,32 @@ class SkeletonApplication extends Application<Configuration> {
     public void initialize(Bootstrap<Configuration> bootstrap) {}
 
     /**
+     * Registers lifecycle managers and Jersey exception mappers
+     *
+     * @param environment
+     * @param buildInfoManager
+     */
+    protected void registerAppManagerLogic(Environment environment,
```

Comment:
registerAppManagerLogic doesn't seem like a great name (it does more than register the lifecycle manager, and 'logic' is superfluous), but i'm not sure i can think of a better one. skeletonSetup? Would it make sense to register the InfoResource endpoint here as well?